waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -240,7 +243,8 @@ genRoutesDir spec =
         ( Just $
             object
               [ "operationsRouteInRootRouter" .= (operationsRouteInRootRouter :: String),
-                "isAuthEnabled" .= (isAuthEnabled spec :: Bool)
+                "isAuthEnabled" .= (isAuthEnabled spec :: Bool),
+                "areApisInUse" .= (not . null $ getApis spec)
```

Comment:
Nitpick: Maybe this is a bit richer?
"inUse" sounds like it is not so much important if they exist, but if they are used. But really we are checking if they exist, right?

```suggestion
                "areThereAnyCustomApiRoutes" .= (not . null $ getApis spec)
```
but if you don't like this ignore it.